frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
```

Comment:
this is just a copy of the card reader not detected screen with the text changed